content/account_management/faq.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Account management FAQ
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+### How do I setup my team in Datadog?
+
+The admin of the account should enter the email addresses of team members
+[here][other-1]. Some team best practices are as follows:
+
+* When the team member receives the confirmation email, they will be provided with a link to log in directly. The user should not click ‘sign up’ during this process.
```

Comment:
Is this last sentence required "The user should not click ‘sign up’ during this process." ?

Reply:
It's a useful precision